### ISSUE TITLE
New tab button is not touch screen friendly

### ISSUE THREAD

**Issue by bsclifton** [brave/browser-laptop#5925]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
If using a touch screen or when on Windows (which is pickier about right click), the new tab button is much harder to do a long press on.  After investigating, it turns out the hitbox is only for the actual SVG itself

**Expected behavior:**
Clicking should be expanded to the size of the parent element or the height of the tabs area

- Platform (Win7, 8, 10? macOS? Linux distro?):
all

- Brave Version:
0.12.10

- Steps to reproduce:
    1. Right click all around the new tab button
    2. Notice that the new tab menu doesn't show up very often, because the hitbox is hard to click

- Screenshot if needed:
Here's a screenshot on Windows. Notice you can make the system menu show up even when you right click within the bounds of the +. It only seems to show if you DIRECTLY right click / long press on the +
![screen shot 2016-11-30 at 1 01 59 am](https://cloud.githubusercontent.com/assets/4733304/20744615/2b9bda02-b69a-11e6-805f-c461ad4372ec.png)

- Any related issues:

**Comment by luixxiul** [comment]
FYI menuButton has the same issue.

**Comment by bradleyrichter** [comment]
We can add a BG box to these if that helps. Just emulate the main toolbar buttons but the style will need to be different. (not white)

**Comment by luixxiul** [comment]
button size can be larger by increasing the width to ~16px if it's better.

**Comment by luixxiul** [comment]
I labelled `QA/no qa needed` as the issue has been superseded with #6125 and PR #6474 for the fix.
